exc2/plot.py

```diff
@@ -0,0 +1,44 @@
+import matplotlib
+from matplotlib.animation import FuncAnimation
+import matplotlib.pyplot as plt
+from matplotlib.widgets import Slider, Button
+import numpy as np
+
+class Animator:
+    def __init__(self, x_min, x_max, dx = 0.001):
+        self.x_array = np.arange(x_min, x_max, dx) #Creating the x axis array
+        self.x_lim = (x_min, x_max)
+        self.fig, self.ax = plt.subplots()
+        self.line, = self.ax.plot([], [], lw=2)
+
+    def function(self, offset = 0):
+        """Calculates the y_array from x_array through
+        the function in question"""
+        lambd = 5*np.sin(2*np.pi*self.x_array)
+        return 3*np.pi*np.exp(-lambd*np.sin(0.01*offset + np.pi/2))

```

Comment:
Did you add more functions for the animated part? :laughing: Because the function that we asked was simpler:

```suggestion
        return 3*np.pi*np.exp(-lambd)

```

Reply:
I now saw `plot_func` and that one has this as I suggested :+1: